apps/web/src/app/(dashboard)/documents/[id]/edit-document.tsx

```diff
@@ -33,6 +33,9 @@ export type EditDocumentFormProps = {
   recipients: Recipient[];
   fields: Field[];
   dataUrl: string;
+  creatorName: string;
+  creatorEmail: string;
+  selfSignExist: boolean;
```

Comment:
Since this is a client component we can pull most of this from the `useSession` hook 😄

Reply:
noted...making a new commit.